ocs_ci/ocs/ui/helpers_ui.py

```diff
@@ -187,3 +187,13 @@ def get_element_type(element_name):
     """
 
     return (f"//a[contains(@title,'{element_name}')]", By.XPATH)
+
+
+def get_element_by_text(text):
+    """
+    This function accepts a text as an argument and returns the element type by creating XPATH for it.
```

Comment:
not critical but the function does not return "the element type" but returns a tuple with locator

Reply:
https://github.com/red-hat-storage/ocs-ci/pull/7993/files#diff-ed3531a3144a61c53aa83362fd1a3f6f15df32919bebaccb49c3b4bc2f69b7ecR181 copied this function